packages/cicero-core/src/template.js

```diff
@@ -174,6 +175,7 @@ class Template {
      * Persists this template to a Cicero Template Archive (cta) file.
      * @param {string} [language] - target language for the archive (should be 'ergo')
      * @param {Object} [options] - JSZip options
+     * @param {Buffer} logo - Bytes data of the PNG file
```

Comment:
This seems inconsistent wit the type check in the `Template` class which uses `string`.